### PR TITLE
Adding RC4 release notes

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: _includes/manuals/1.3/1.2_release_notes.md [hunk 1]
```diff
@@ -174,6 +175,7 @@ manual section.
 - Added NIC CRUD, power and boot operations ([#3046](http://projects.theforeman.org/issues/3046))
 - API v1/v2 StatisticsController and removed JSON from UI controller ([#2984](http://projects.theforeman.org/issues/2984))
 - Route /api/fact_values was missing from v2 API ([#3165](http://projects.theforeman.org/issues/3165))
+- Extra validations fo V2 API ([#3107](http://projects.theforeman.org/issues/3107))
```
Comment (domcleal) [inline _includes/manuals/1.3/1.2_release_notes.md]: 'for'

Comment (domcleal) [inline _includes/manuals/1.3/1.2_release_notes.md]: Should be 3207?  If so, it's removing additional (useless) validation.

File: _includes/manuals/1.3/1.2_release_notes.md [hunk 2]
```diff
@@ -253,6 +255,10 @@ manual section.
 - Package rubygem-ancestry was conflicting with f19-updates ([#3142](http://projects.theforeman.org/issues/3142))
 - Fixed incompatible gettext_i18n_rails with gettext 2.x ([#3167](http://projects.theforeman.org/issues/3167))
 - We now print message at the end of configuration ([#3131](http://projects.theforeman.org/issues/3131))
+- Configuration ssldir was fixed for Puppet ([#3173](http://projects.theforeman.org/issues/3173))
```
Comment (domcleal) [inline _includes/manuals/1.3/1.2_release_notes.md]: Proxy section?

File: _includes/manuals/1.3/1.2_release_notes.md [hunk 3]
```diff
@@ -299,6 +305,11 @@ manual section.
 - Avoided flapping os.release_name for Debian ([#2244](http://projects.theforeman.org/issues/2244))
 - Fixed duplicate Organization tabs ([#3139](http://projects.theforeman.org/issues/3139))
 - Fixed undefined local variable or method render_403 ([#3141](http://projects.theforeman.org/issues/3141))
+- Settings descriptions were not updating ([#2756](http://projects.theforeman.org/issues/2756))
+- Array values in settings UI were not displayed correctly after update ([#3180](http://projects.theforeman.org/issues/3180))
+- Fixed assignement of a Security group to EC2 instance when creating host ([#3133](http://projects.theforeman.org/issues/3133))
```
Comment (domcleal) [inline _includes/manuals/1.3/1.2_release_notes.md]: 'assignment'

**Comment by domcleal** [comment]
:+1:

**Comment by lzap** [comment]
Pushed with 2efd6a5 thanks!

**Pull request merged**